cli/asc.js

```diff
@@ -342,12 +369,74 @@ exports.main = function main(argv, options, callback) {
           }
         }
       }
+      /*
+      In this case the library wasn't found so we check paths
+      */
+      if (sourceText == null) {
+        if (args.traceResolution) {
+            stderr.write("Looking for " + sourcePath + " imported by " + dependee + " " + EOL);
+        }
+        paths = getPaths(path.join(baseDir, dependee));
+        let _package = sourcePath.replace(/\~lib\/([^\/]*).*/, "$1");
+        for (let _path of paths) {
+          if (args.traceResolution) {
+            stderr.write(`    in ${_path}`);
```

Comment:
I guess if everywhere use ordinal string concatenation we should follow this convention. Or refactor rest parts with using string template literals